packages/performance-tests/test/identity-transformer.ts

```diff
@@ -0,0 +1,79 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as path from "path";
+import * as fs from "fs";
+import { Element, IModelHost, IModelHostConfiguration, Relationship, SnapshotDb, BriefcaseDb } from "@itwin/core-backend";
+import { Logger, LogLevel, PromiseReturnType, StopWatch } from "@itwin/core-bentley";
+import { IModelTransformer, TransformerLoggerCategory } from "@itwin/imodel-transformer";
+//import { TestBrowserAuthorizationClient } from "@itwin/oidc-signin-tool";
+import { TestIModel } from "./TestContext";
+import { Reporter } from "@itwin/perf-tools";
+import { initOutputFile, timed } from "./TestUtils";
+
+/*
+ * Tests where we perform "identity" transforms, that is just rebuilding an entire identical iModel (minus IDs)
+ * through the transformation process.
+ */
+
+const loggerCategory = "Transformer Performance Tests Identity";
+const outputDir = path.join(__dirname, ".output");
+
+export default async function identityTransformer(iModel: TestIModel, os: any, reporter: Reporter){
```

Comment:
```suggestion
export default async function identityTransform(iModel: TestIModel, reporter: Reporter){
```

rename and just `import * as os from "os";`, no need to take it as an argument, it's a standard import